playground/src/assembleDecorations.ts

```diff
@@ -0,0 +1,156 @@
+import * as tsdoc from '@microsoft/tsdoc';
```

Comment:
>import [](start = 0, length = 6)

Can we convert this to a class, e.g. `TSDocColorizer` or `DocCommentSyntaxHighlighter` or somesuch?